### PR TITLE
docs: gRPC Access Log Service (ALS) sink

### PR DESCRIPTION
**Pull request by zirain**
fix: https://github.com/envoyproxy/gateway/issues/3745

### CODE REVIEW

File: site/content/en/latest/tasks/observability/proxy-accesslog.md [hunk 1]
```diff
@@ -140,6 +140,61 @@ Verify logs from loki:
...
+  name: als
+  namespace: envoy-gateway-system
+spec:
+  telemetry:
+    accessLog:
+      settings:
+        - format:
+            type: Text
+            text: |
+              [%START_TIME%] "%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% "%REQ(X-FORWARDED-FOR)%" "%REQ(USER-AGENT)%" "%REQ(X-REQUEST-ID)%" "%REQ(:AUTHORITY)%" "%UPSTREAM_HOST%"
```
Comment (zirain) [inline site/content/en/latest/tasks/observability/proxy-accesslog.md:180]: wait https://github.com/envoyproxy/gateway/pull/3770

**Review by shawnh2 — APPROVED** [review]
lgtm

**Review by Xunzhuo — DISMISSED** [review]

**Comment by zirain** [comment]
wait https://github.com/envoyproxy/gateway/pull/3818

**Review by AliceProxy — APPROVED** [review]

**Pull request merged**